src/Event/UnitsRefunded.php

```diff
@@ -11,21 +11,13 @@
 {
     use PayloadTrait;
 
-    /** @var string */
-    private $orderNumber;
-
-    /** @var iterable */
-    private $unitIds;
-
-    /** @var int */
-    private $amount;
-
-    public function __construct(string $orderNumber, iterable $unitIds, int $amount)
+    public function __construct(string $orderNumber, iterable $unitIds, iterable $shipmentIds, int $amount)
```

Comment:
To be honest, when it comes to class naming, the difference between `UnitRefunded` and `UnitsRefunded` is so slight that it took me a few moments to figure out what's going on. WDYT about `OrderUnitsRefunded` or something that will let everyone easily distinguish between these two events?

Reply:
For me the perfect naming would be `RefundLineItems` as both order items and shipments are LineItems in the context of Invoice and the result of a refund is Credit Memo creation. However, since we want to separate RefundPlugin and InvoicingPlugin from each other, I think your idea for renaming these events is just fine and acceptable :)

Reply:
cc @GSadee @pamil I would like to see your opinion 😄

Reply:
For me, `OrderUnitRefunded` is even more confusing than `UnitRefunded`, because we don't have an `OrderUnit`, but `OrderItemUnit` 🎉 

And `RefundLineItems` is not a best solution imo, because we don't have this concept in this plugin.

To be honest, none of the proposed changes convinces me, I would be for leaving that as it is now.

Reply:
Ok, so to not make it longer - I would vote for merging it as it is and I'll open an issue about that naming, ok? @bartoszpietrzak1994 @GSadee

Reply:
I think it's okay right now as it is. They only thing I would change in this line of code is `iterable` to `array` (as event payload should be serializable, and `iterable` here is way too general).